out of restart_dir again
test 3
